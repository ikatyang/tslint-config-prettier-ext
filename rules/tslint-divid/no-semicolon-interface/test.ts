interface Foo {
  bar: string,
  zoo(): number,
}
