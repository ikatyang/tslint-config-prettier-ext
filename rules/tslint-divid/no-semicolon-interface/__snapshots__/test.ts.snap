// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
no error

======
ERROR: (no-semicolon-interface) /rules/tslint-divid/no-semicolon-interface/test.ts[2, 3]: Use comma instead of semicolon in interfaces.
ERROR: (no-semicolon-interface) /rules/tslint-divid/no-semicolon-interface/test.ts[3, 3]: Use comma instead of semicolon in interfaces.

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
interface Foo {
  bar: string,
  zoo(): number,
}

======
interface Foo {
  bar: string;
  zoo(): number;
}

>>>>>> after
"
`;
